language/llama3-405b/run_accuracy.sh

```diff
@@ -0,0 +1,22 @@
+CHECKPOINT_PATH="${CHECKPOINT_PATH:-meta-llama/Llama-2-70b-chat-hf}"
```

Comment:
Modle name and sample code - to fix for all .sh script

Reply:
Done